### ISSUE TITLE
Create executables for the CLI with PyInstaller

### ISSUE THREAD

**Issue by balazsdukai** [cityjson/cjio#41]
cjio_dbexport has a Travis build+deploy setup, we can use the same. Having a single file executable could help a lot in adoption, since the user doesn't even need to have python installed. For linux, mac, windows

**Comment by hugoledoux** [comment]
yeah but our software is CLI... If you can manage CLI you should be able to pip install, no?

But that would solve issues with versions (of like pyproj...) so yeah nice!

**Comment by balazsdukai** [comment]
I'm especially thinking about Windows users here. Pip on Unix is a piece of cake, on Windows it can be problematic, especially with cpp dependencies. 

And yeah, I thought the same about CLI and being able to manage pip, but lately I've been proven wrong...
